utils/cq/requirements.txt

```diff
@@ -7,7 +7,7 @@ clustershell
 ## flake8 6 removed --diff option which breaks flake precommit hook.
 ## https://github.com/pycqa/flake8/issues/1389 https://github.com/PyCQA/flake8/pull/1720
 flake8<6.0.0
-isort
+isort==5.9
```

Comment:
@YinglinSun Do you see any issues with pinning this? It ensures everyone uses the same check. Then, when we have a GHA we can pin to the same to avoid discrepancies

Reply:
No concern. Should be good idea.

Reply:
Why 5.9 rather than 5.12 and what's the specific reason here?  I have an idea that I'm working on for making sure that versions are up-to-date and I'd rather address everything at the same time for simplicity.

Reply:
5.9 only because that's what @YinglinSun initially used. If we go higher, we just need to make sure it's available on supported distros without special install

Reply:
Or, we need to make sure a higher version doesn't introduce new checks, and maybe pin a range or `>=`

Reply:
We shouldn't base this on the lower common denominator for all possible distros that developers could be using, if a developer wants a specific version they should pip install it (and they should probably be doing that anyway).  When pinning a version we should use the most recent version or have a reason not to and a plan to resolve it, plus we should have a process for updating the version over time.